src/app_ui.py

```diff
@@ -142,15 +142,21 @@ def create_new(self):
                     reference_type = "phdthesis"
                 case "6":
                     return
-                case default:  # pylint: disable=unused-variable
+                case _:
```

Comment:
Extra: The default case doesn't use the input value, so I replaced it with a regular wildcard.